src/vm/jitinterface.cpp

```diff
@@ -3146,6 +3146,8 @@ void CEEInfo::ComputeRuntimeLookupForSharedGenericToken(DictionaryEntryKind entr
     CORINFO_RUNTIME_LOOKUP *pResult = &pResultLookup->runtimeLookup;
     pResult->signature = NULL;
 
+    pResult->indirectFirstOffset = 0;
```

Comment:
I think there are several more places, where the variable should be initialized.